pkg/dcgmexporter/expcollector.go

```diff
@@ -217,8 +218,8 @@ func newExpCollector(
 	var labelsCounters []Counter
 	for i := 0; i < len(counters); i++ {
 		if counters[i].PromType == "label" {
-			labelsCounters = append(labelsCounters, counters[i])
-			counters = slices.Delete(counters, i, i+1)
+			counter := counters[i]
+			labelsCounters = append(labelsCounters, counter)
```

Comment:
This can be:
```
labelsCounters = append(labelsCounters, counters[i])
```

This avoid creating `counter`.

Reply:
done!